recipe-server/client/control_new/less/main.less

```diff
@@ -113,9 +155,8 @@ body {
 
 /* Buttons at the bottom of a form */
 .form-actions {
-  background: #FBFBFB;
-  border: 1px solid #D9D9D9;
-  padding: 15px;
+  border-top: 1px solid #eee;
```

Comment:
Linter should be complaining about `#eee` being lowercase. Double check that the npm script is running against `less` files.